src/restful/controllers/applicationsController.js

```diff
@@ -134,12 +134,10 @@ class ApplicationsController {
         await applicationRef.update({ status });
 
         if (status === "accepted") {
-            const roomId = uuidv4();
+            const roomId = 'chat_'+(creator_id < brand_id? creator_id+"_"+brand_id : brand_id+"_"+creator_id)
```

Comment:
Consistency: If two users are identified by creator_id and brand_id, the room ID should be the same whether one user creates the room or the other user does. Without this conditional, you could end up with two different room IDs for the same pair of users, depending on the order of the IDs.

Example
**Assume creator_id is 5 and brand_id is 8.**

With the conditional: Since 5 < 8, the roomId will be chat_5_8.
If the roles were reversed and creator_id was 8 and brand_id was 5, the roomId would still be chat_5_8 because of the conditional ordering.

Reply:
@Kaleshe

Reply:
A room will automatically be created once a Brand accepts an application. A room can not be created manually. So that means we can assume it will always be in the order brand_id + creator_id

Reply:
We should also probably add a check to the createRoom function, to see if a room already exists between the 2 users. If it does it should return something like this

```
    return {
      status: 200,
      roomId: roomId,
      room: roomData
    };
```